MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -170,12 +170,12 @@ class RouteManeuverViewController: UIViewController {
     }
     
     func updateStreetNameForStep() {
-        if let step = step, step.isNumberedMotorway, let codes = step.codes {
+        if let destinations = step?.destinations {
+            destinationLabel.unabridgedText = destinations.joined(separator: NSLocalizedString("DESTINATION_DELIMITER", bundle: .mapboxNavigation, value: " / ", comment: "Delimiter between destinations in a road concurrency"))
```

Comment:
We should still find a way to ensure that a destination doesn’t get cut off in the middle, like “Stevens Creek Blvd / West San ~~Carlos Street~~”.